enterprise/server/remote_execution/container/container.go

```diff
@@ -392,7 +298,7 @@ func TrackStats(ctx context.Context, c CommandContainer) (stop func(), res <-cha
 			// the container, which can take a few hundred ms or possibly longer
```

Comment:
the mention of podman on the line above might be out of date.

Reply:
That part is still accurate, since podman doesn't (yet) have the logic where we create the cgroup then run the container in that cgroup that we created. Instead, podman manages the cgroup lifecycle itself.

I removed this note for now though, since it's not super important. The bigger TODO is to make sure that the container implementations don't delete the cgroup from under us (this is a problem with both ociruntime and podman). That way, we could remove this polling, and reliably read the cgroup stats once at the end of execution. I think we could do this by creating a child cgroup under the container cgroup and run oci/podman containers in the child, so that even after crun/podman delete the child cgroup, we can still read the stats from the parent cgroup.